app/lib/constituency_api.rb

```diff
@@ -1,7 +1,14 @@
 module ConstituencyApi
   ConstituencyApiError = Class.new(RuntimeError)
 
-  class Constituency < Struct.new(:name)
+  class Constituency < Struct.new(:name, :mp)
+  end
+
+  class Mp < Struct.new(:id, :name, :start_date)
+    URL = "http://www.parliament.uk/biographies/commons"
+    def url
+      "#{URL}/#{name.gsub(/\s+/, "-")}/#{id}"
```

Comment:
I think we should be more restrictive when constructing the URL, substituting whitespace might not be enough and we might end up having invalid URLs (if MP's name includes UTF8 characters).

We could probably use this: http://api.rubyonrails.org/classes/ActiveSupport/Inflector.html#method-i-parameterize